### DISCUSSION TITLE
服务名无效。 请键入 NET HELPMSG 2185 以获得更多的帮助

### DISCUSSION THREAD

**Discussion by vipnet2025** [XIU2/CloudflareSpeedTest#446] [讨论]
### 问题描述

下载的CloudflareST_windows_amd64.zip 使用cfst_3proxy.bat 测速时无法替换最优化IP地址，提示服务名无效。 请键入 NET HELPMSG 2185 以获得更多的帮助

### 软件版本

CloudflareST_windows_amd64.zip

**Comment by XIU2** [comment]
这不是 CloudflareST 的问题，运行之前请先搞清楚 `cfst_3proxy.bat` 是干什么的。
